docs/deployment/landing_page.rst

```diff
@@ -79,17 +79,17 @@ signature:
 
     openssl req -new -newkey rsa:4096 -nodes -sha256 -keyout domain.com.key -out domain.com.csr
 
-**Don't load any resources (scripts, web fonts, etc.) from 3rd parties
+**Don't load any resources (scripts, web fonts, etc.) from third parties
```

Comment:
third-parties ?

Reply:
"Third" is an adjective modifying "parties", here.